pages/Soulbound-Weapons.md

```diff
@@ -6,4 +6,4 @@ They are an unenchanted Diamond Sword and a Bow, respectively, and may be enchan
 * [Soulbound Tools](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Soulbound-Tools)
 * [Soulbound Armor](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Soulbound-Armor)
 * [Soulbound Backpack](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Soulbound-Backpack)
-* [Soulbound Rune](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Soulbound-Rune)
+* [Soulbound Rune](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ancient-Rune-[Soulbound])
```

Comment:
Don't use brackets in the URL...
Please revert this and rename the page to "Soulbound-Rune"